### DISCUSSION TITLE
How to convert a mask-image: linear gradient to tailwind class?

### DISCUSSION THREAD

**Discussion by jaeyson** [tailwindlabs/tailwindcss#9417] [Help] [answered]
Hi! I have the following css that I'd like to move this to a class, how would I do this?

```css
-webkit-mask-image: -webkit-gradient(linear, 0% 0%, 0% 100%, from(rgb(0, 0, 0)), to(rgba(0, 0, 0, 0)));
```

Any help is greatly appreciated!

**Comment by wongjn** [comment, marked as answer]
You could use an [arbitrary property](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-properties) class: `[-webkit-mask-image:-webkit-gradient(linear,0%_0%,_0%_100%,from(rgb(0,0,0)),to(rgba(0,0,0,0)))]`

**Reply by jaeyson** [reply]
Ah that helped! especially the square bracket wrapped around the property. nifty!
